recipes/openmm/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- 10.0
```

Comment:
Do you support other CUDA versions? Or only 10.0?

Also do you want to have a CPU only build? Or are you only interested in building with GPU support?

Reply:
We will add more CUDA versions, yes. I think we can do it [like the ucx recipe](https://github.com/conda-forge/ucx-split-feedstock/blob/master/recipe/conda_build_config.yaml), right?

OpenMM will just ignore CUDA/OpenCL if the platform is not compatible, so we don't _need_ CPU-only packages, I'd say. We might be interested in building variants without the explicit `cudatoolkit` runtime dependency though, so people can use their own CUDA installation without downloading a several hundreds MB package. How are you handling this scenario right now?

Reply:
By adding a [`None` version]( https://github.com/conda-forge/ucx-split-feedstock/blob/e807987cd0a8d36ffdd2d4edb41c80442cef8b17/recipe/conda_build_config.yaml#L4 ) and [a selector to disable the CUDA compiler in that case]( https://github.com/conda-forge/ucx-split-feedstock/blob/e807987cd0a8d36ffdd2d4edb41c80442cef8b17/recipe/meta.yaml#L50 ).

Reply:
But we still want to build the CUDA libraries in our package. We just want to ignore `cudatoolkit` for one of the variants so the user provides their own CUDA install (like in a cluster).

Reply:
Ah. I'm not sure that we have a good way to solve this ATM. What is gained by using an existing CUDA install vs. using the `cudatoolkit` package?

Reply:
I'd say there are two main reasons:

1. Avoiding the large `cudatoolkit` download.
2. Allowing HPC sysadmins to provide their own CUDA environment.

Maybe @peastman wants to add something else.

Reply:
That sounds right.

Reply:
Please raise an issue on the [`nvcc` issue tracker]( https://github.com/conda-forge/nvcc-feedstock/issues ). We can discuss this further there.

Reply:
Done! https://github.com/conda-forge/nvcc-feedstock/issues/14